tasks/prime.py

```diff
@@ -333,6 +333,30 @@ def update_mto_service_item(self):
         if success:
             self.replace_prime_data(PrimeObjects.MTO_SERVICE_ITEM, mto_service_item, resp)
 
+    @tag(PrimeObjects.POST_COUNSELING_INFORMATION.value, "updateMTOPostCounselingInformation")
+    def update_post_counseling_information(self):
```

Comment:
The reason this task isn't showing up is because you need to give this function the `@task` decorator - super easy to miss ☺️